plugins/net_plugin/net_plugin.cpp

```diff
@@ -47,7 +62,7 @@ namespace eos {
     transaction_id_type id;
     bool                is_known_by_peer = false; ///< true if we sent or received this trx to this peer or received notice from peer
     bool                is_noticed_to_peer = false; ///< have we sent peer notice we know it (true if we receive from this peer)
-    uint32_t            block_num = -1; ///< the block number the transaction was included in
+    UInt16              block_num = -1; ///< the block number the transaction was included in
```

Comment:
Why is this 16 bit?

Reply:
because that's what was in the generated Transaction  type.  See 
libraries/types/include/.../generated.hpp